### PR TITLE
Added tests for simple constructions via .yaml format

### PR DESCRIPTION
**Pull request by AndrewG0R**

### CODE REVIEW

File: src/test/scala/org/polystat/py2eo/Commons.scala [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package org.polystat.py2eo
+
+import org.junit.Assert.assertTrue
+import org.polystat.py2eo.Main.{debugPrinter, writeFile}
+
+import java.io.File
+import scala.sys.process._
+
+trait Commons {
+  val testsPrefix: String = System.getProperty("user.dir") + "/src/test/resources/org/polystat/py2eo/"
```
Comment (PetrB09) [inline src/test/scala/org/polystat/py2eo/Commons.scala:10]: Why not loading resourses from classpath?

File: src/test/scala/org/polystat/py2eo/Tests.scala [hunk 2]
```diff
@@ -2,33 +2,25 @@ package org.polystat.py2eo
...
 import scala.language.postfixOps
 import scala.sys.process._
 
-object SlowTest extends Tag("SlowTest")
 class YamlItem(var testName: Path, var yaml: java.util.Map[String, Any])
 
 class Tests {
-  val separator: String = "/"
   var files = Array.empty[File]
   private val testsPrefix = System.getProperty("user.dir") + "/src/test/resources/org/polystat/py2eo/"
```
Comment (PetrB09) [inline src/test/scala/org/polystat/py2eo/Tests.scala:21]: see above

**Review by PetrB09 — APPROVED** [review]

File: src/test/scala/org/polystat/py2eo/Commons.scala (hunk 1 above)
Comment (AndrewG0R) [reply src/test/scala/org/polystat/py2eo/Commons.scala:10]: resolved

File: src/test/scala/org/polystat/py2eo/Tests.scala (hunk 2 above)
Comment (AndrewG0R) [reply src/test/scala/org/polystat/py2eo/Tests.scala:21]: resolved

**Pull request closed without merging**